tests/AbstractProcessTest.php

```diff
@@ -299,6 +317,49 @@ public function testTerminateWithStopAndContinueSignalsUsingEventLoop()
     }
 
     /**
+     * @dataProvider provideOutputSizeAndExpectedMaxDuration
+     */
+    public function testProcessWithFixedOutputSize($size, $expectedMaxDuration = 5)
+    {
+        // Note: very strange behaviour of Windows (PHP 5.5.6):
+        // on a 1000 long string, Windows succeeds.
+        // on a 10000 long string, Windows fails to output anything.
+        // On a 100000 long string, it takes a lot of time but succeeds.
```

Comment:
@jmikola: Yes, this test fails on Windows... In PHP > 5.6.3, it fails. In PHP < 5.6.3, it locks and the test never halts, which is even worst. #11 fixes this test case.